### PR TITLE
Update AboutUs

### PR DESCRIPTION
**Pull request by YuZizhen**
Update AboutUs file to include some information about Zizhen.

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -7,48 +7,46 @@ We are a team based in the [School of Computing, National University of Singapor
...
 ### Koh Guan Zeh
 
 <img src="images/kohguanzeh.png" width="200px">
 
 [[github](https://github.com/KohGuanZeh)]
 
-* Role: Developer
-* Responsibilities: Testing
+- Role: Developer
+- Responsibilities: Testing
```
Comment (Javiery3889) [inline docs/AboutUs.md:25]: I think your editor auto-formatted the markdown bullet point to from `*` to `-`, let us standardise it to `*` for all markdown documents

File: docs/AboutUs.md [hunk 2]
```diff
@@ -7,48 +7,46 @@ We are a team based in the [School of Computing, National University of Singapor
...
-<img src="images/johndoe.png" width="200px">
+<img src="images/yuzizhen.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/YuZizhen)]
 
-* Role: Project Advisor
+- Role: Developer
```
Comment (Javiery3889) [inline docs/AboutUs.md:17]: Do add your responsibility as well

**Review by Javiery3889 — CHANGES_REQUESTED** [review]

File: docs/AboutUs.md (hunk 1 above)
Comment (YuZizhen) [reply docs/AboutUs.md:25]: ok ok just see this! thanks

File: docs/AboutUs.md (hunk 2 above)
Comment (YuZizhen) [reply docs/AboutUs.md]: Noted with thanks!!

**Comment by Javiery3889** [comment]
@YuZizhen Please close this PR as you are no longer using this branch

**Pull request closed without merging**